frontend/src/index.jade

```diff
@@ -90,7 +90,9 @@ html
             .summary-chart(v-for="(user, i) in repo")
               .summary-chart__title(
                 title="click to view author's code",
-                v-on:click="$emit('view-authorship', {author:user.name, repo:user.repoName, name:user.displayName})"
+                v-on:click=
+                  "$emit('view-authorship', " +
+                  "{author:user.name, repo:user.repoName, name:user.displayName, minDate:minDate, maxDate:maxDate})"
```

Comment:
the `+` operator should  be on this line, also line wrap should use double the indentation from previous line.